<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NUMERO 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NUMERO 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RESULTADO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtN1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtN2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRes">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSuma">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SUMA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResta">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RESTA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultiplicar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>MULTIPLICAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDividir">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DIVIDIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
